api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -176,23 +175,57 @@ public function validateConversionProfile(entry $sourceObject = null)
 	 */
 	public function validateForUpdate($sourceObject, $propertiesToSkip = array())
 	{
-		/* @var $sourceObject LiveEntry */
-		if(isset($this->dvrStatus))
+		$stopedOnlyUpdateAttributes = array(
+				"dvrStatus" => array("validatePropertyChanged"), 
+				"dvrWindow" => array("validatePropertyChanged"), 
+				"recordStatus" => array("validatePropertyChanged","validateRecordedEntryId"), 
+				"conversionProfileId" => array("validatePropertyChanged","validateRecordedEntryId")
```

Comment:
It can't be insert-only anymore, if user can add recording he might need to set the options as well.